app/helpers/duplicate_intake_guard.rb

```diff
@@ -1,7 +1,6 @@
 class DuplicateIntakeGuard < SimpleDelegator
   def has_duplicate?
-    Intake::GyrIntake
-      .where(primary_consented_to_service: "yes")
+    ClientLoginService.new(:gyr).active_intake_accounts
```

Comment:
This shouldn't conflict with my work. I'm working in this file right now but will be changing the second part of this query. Removing the check for matching `email_address` or `phone_numbers` and matching for `primary_ssn` instead.